### PR TITLE
Allow charging schedule configuration

### PR DESCRIPTION
**Pull request by nanomad**
Fixes https://github.com/cliviu74/wallbox/issues/38

### CODE REVIEW

**Review by cliviu74 — CHANGES_REQUESTED** [review]
Can you please document getChargerSchedules and setChargerSchedules methods in the main README.md?

File: wallbox/wallbox.py [hunk 1]
```diff
@@ -190,3 +190,49 @@ def setEnergyCost(self, chargerId, energyCost):
...
+            response = requests.get(
+                f"{self.baseUrl}chargers/{chargerId}/schedules", headers=self.headers,
+                timeout=self._requestGetTimeout
+            )
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as err:
+            raise (err)
+        return json.loads(response.text)
+
+    """
```
Comment (cliviu74) [inline wallbox/wallbox.py:205]: Can you please document this method in the README.md?

**Comment by nanomad** [comment]
@cliviu74 done :)

**Comment by cliviu74** [comment]
Nice one, thanks a million @nanomad

**Pull request merged**